TASK: Neos 9 Compatibility

adrian-cerdeira opened the pull request:
I used rector to upgrade the code, but it fails in some parts in an neos 9 instance this package:
`Data source with identifier "tms-select-nodedata" does not exist. - Reference code "202507100826581a2e23"`
Can somebody check this and resolve it.
Package can be installed in an Neos 9 Instance it was already added. But right now it doesnt work.

Commit:
TASK: Neos 9 Compatibility via rector

jobee commented:
Hi @adrian-cerdeira,
I won't update this package for Neos 9 and I am the only maintainer right now.
If you or others still see value in this, we can discuss transferring the repository.

adrian-cerdeira commented:
> Hi @adrian-cerdeira, I won't update this package for Neos 9 and I am the only maintainer right now. If you or others still see value in this, we can discuss transferring the repository.

Okay. I need to discuss this with my company, because of the transferring or if we really need this package.
I would find it very nice to have it as package. I am gonna inform you about next steps.

Commit:
TASK: remove caching and nodesignal to rewrite for event sourcing

Commit:
feat: resolve neos 9 todos to make it compatibel

Commit:
chore: update readme to remove disable cache (cache dont exist anymore)

adrian-cerdeira commented:
@jobee Hello, 
We have discussed it. 
I updated the package to be working and be compatibel with Neos 9.
Some stuff like caching and more who has changed completely in Neos 9 have I removed.
It works now fine in Neos 9. If you want you can create a new neos 9 release or transfer it to me.
What do you need for the transfer?
[My Email](mailto:adrian.cerdeira@ongoing.ch)
Thank you 🙏

Commit:
feat: renable caching and integrate command hook for replacement of node signal

adrian-cerdeira commented:
@paavo caching and flush of caching works now fine. You can now review this PR.

Commit:
fix: renaming also in factory

paavo commented:
@jobee as discussed, this PR seems to be working 🎉

jobee commented:
Thanks for your work! I can't test it on Neos 9 right now, but I trust you. Feel free to merge and tag the commit with `2.0.0` as it obviously contains breaking changes.
